pcsx2/VMManager.cpp

```diff
@@ -2981,7 +2982,13 @@ void VMManager::WarnAboutUnsafeSettings()
 	if (EmuConfig.GS.AccurateBlendingUnit <= AccBlendLevel::Minimum)
 	{
 		append(ICON_FA_BLENDER,
-			TRANSLATE_SV("VMManager", "Blending is below basic, this may break effects in some games."));
+			TRANSLATE_SV("VMManager", "Current Blending Accuracy is below Basic, this may break effects in some games."));
+	}
+	if (EmuConfig.GS.AccurateBlendingUnit == AccBlendLevel::Maximum)
```

Comment:
This is kinda useless, other options may be slow as well, or if the gpu supports fbfetch there's barely any impact.

It was fine as it was.

Reply:
The idea is to let the user know that higher blending settings are heavy, especially for lower-end GPUs (or AMD GPUs).

I suppose the warning could be placed lower, so that it shows up when blending is above High instead of at Maximum only.